extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamInfoItemExtractor.java

```diff
@@ -324,4 +324,16 @@ public String getShortDescription() throws ParsingException {
 
         return null;
     }
+
+    @Override
+    public boolean isShortFormContent() throws ParsingException {
+        try {
+            final String webPageType = videoInfo.getObject("navigationEndpoint")
+                    .getObject("commandMetadata").getObject("webCommandMetadata")
+                    .getString("webPageType");
+            return !isNullOrEmpty(webPageType) && webPageType.equals("WEB_PAGE_TYPE_SHORTS");
+        } catch (final Exception e) {
+            throw new ParsingException("Could not determine if short-form content", e);
```

Comment:
This seems not understandable as an exception message: I suggest using something like this (given by @opusforlife2 to me):

```suggestion
            throw new ParsingException("Could not determine if this is short-form content", e);
```